massa-models/src/node_configuration/default_testing.rs

```diff
@@ -111,18 +111,22 @@ pub const T0: MassaTime = MassaTime::from(32000);
 * };
 */
 
-pub const DISABLE_BLOCK_CREATION: bool = true;
-pub const FORCE_KEEP_FINAL_PERIOD: u64 = 0;
-pub const FUTURE_BLOCK_PROCESSING_MAX_PERIODS: u64 = 10;
-pub const LEDGER_CACHE_CAPACITY: u64 = 1_000_000;
-pub const LEDGER_RESET_AT_STARTUP: bool = true;
-pub const MAX_DEPENDENCY_BLOCK: usize = 10;
-pub const MAX_DISCARED_BLOCKS: usize = 10;
-pub const MAX_FUTURE_PROCESSING_BLOCK: usize = 10;
-pub const MAX_ITEM_RETURN_COUNT: usize = 1000;
-pub const MAX_OPERATION_FILL_ATTEMPTS: u32 = 6;
-pub const OPERATION_BATCH_SIZE: usize = 3;
-pub const POS_DRAW_CACHED_CYCLE: usize = 10;
+pub const CURSOR_DELAY: MassaTime = MassaTime::from(0); // normally in config.toml
```

Comment:
I'm not sure “normally” is a word that I understand without ambiguity, do you mean “by default”, “previously” or “without modification in test mode”?

Reply:
The explanation of the usage of all that one are just above in a developer note. It's just seems that no one read it. That comments could be "have you read the notes above?"

Reply:
>  "have you read the notes above?"

This seems perfect ! :p